google2018/ocr_is_cool/requirements.txt

```diff
@@ -42,7 +42,7 @@ lxml==4.2.1
 Markdown==3.0
 MarkupSafe==1.0
```

Comment:
Only hh1 has authority over herself and jj chris nick any of them everyone is blocked from them.